src/cljs/main/broadfcui/page/method_repo/method_config_importer.cljs

```diff
@@ -106,12 +106,10 @@
   {:render
    (fn [{:keys [props state this]}]
      (cond
-       (and (:loaded-config @state)
-         (or (:workspace-id props) (:workspaces-list @state)))
-       (create-import-form state props this (:loaded-config @state) true
-         [{:label "Configuration Namespace" :key "namespace"}
-          {:label "Configuration Name" :key "name"}])
-
+       (and (:loaded-config @state) (or (:workspace-id props) (:workspaces-list @state)))
```

Comment:
This indentation pattern feels more logical to me.  Please let me know if I am violating some standard here.